docker/Dockerfile

```diff
@@ -6,7 +6,10 @@ USER root
 COPY requirements.txt /appgate
 
 RUN apt-get update && \
-    apt-get install git --yes && \
+    apt-get install --yes \
+        git \
+        # CVE-2023-5981
+        libgnutls30=3.7.9-2+deb12u1 && \
```

Comment:
Do we need a TODO here for when we can remove this?

Reply:
Adding TODO